packages/components/color-picker/src/composables/use-alpha-slider.ts

```diff
@@ -62,11 +67,38 @@ export const useAlphaSlider = (props: AlphaSliderProps) => {
     }
   }
 
+  function handleKeydown(event: KeyboardEvent) {
+    const step = 1
+    switch (event.code) {
+      case EVENT_CODE.left:
+      case EVENT_CODE.down:
+        event.preventDefault()
+        event.stopPropagation()
+        incrementPosition(-step)
+        break
+      case EVENT_CODE.right:
+      case EVENT_CODE.up:
+        event.preventDefault()
+        event.stopPropagation()
```

Comment:
This is repeated with lines 75 and 76, so it may be written at the front.

Reply:
This is mainly to avoid affecting other keys, such as `Enter` (In the future, it will be used to select colors and close the dropdown)

Reply:
Ok.